frontend/src/modules/organization/helpers/organization.helpers.ts

```diff
@@ -34,22 +34,20 @@ const useOrganizationHelpers = () => {
   const emails = (organization: Organization) => (organization.emails || []);
 
   const primaryDomains = (organization: Organization) => organization.identities
-    .filter((i) => OrganizationIdentityType.PRIMARY_DOMAIN === i.type);
+    .filter((i) => OrganizationIdentityType.PRIMARY_DOMAIN === i.type && !['email'].includes(i.platform));
 
   const alternativeDomains = (organization: Organization) => organization.identities
-    .filter((i) => OrganizationIdentityType.ALTERNATIVE_DOMAIN === i.type);
+    .filter((i) => OrganizationIdentityType.ALTERNATIVE_DOMAIN === i.type && !['email'].includes(i.platform));
```

Comment:
I don't think this is ok, the problem is not the platform that's an email, it's with identities with type email
Check this organization as an example https://cm.lfx.dev/organizations/c05b45d0-e458-419d-b609-95ad8e47a429?projectGroup=dc48fac5-b31a-4659-ac99-60eb52a1082a&segmentId=dc48fac5-b31a-4659-ac99-60eb52a1082a

Also in line 34, you are still getting the emails from the wrong place. It should be from identities with type email

Reply:
I think you can remove the `!['email'].includes(i.platform))` from everywhere on this file. This will be excluding all valid identities such as domains that came from emails